lib/pengine/utils.c

```diff
@@ -834,10 +834,17 @@ unpack_operation(action_t * action, xmlNode * xml_obj, resource_t * container,
         } else {
             value = "recover baremetal remote node connection (default)";
         }
-        if (action->rsc->remote_reconnect_interval) {
+
+        if ((action->rsc->remote_reconnect_interval) ||
+            !is_set(action->rsc->flags, pe_rsc_managed)) {
             action->fail_role = RSC_ROLE_STOPPED;
         }
-        action->on_fail = action_fail_reset_remote;
+
+        if (is_set(action->rsc->flags, pe_rsc_managed)) {
+            action->on_fail = action_fail_reset_remote;
+        } else {
+            action->on_fail = action_fail_stop;
+        }
```

Comment:
Probably you can always find a config that screws it up but here I guess it is safer to
overrule with action_fail_stop to prevent totally unexpected behaviour.